packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -27,8 +29,10 @@ export function getMomentTemplateBanner(
         numArticles,
         onCtaClick,
         onSecondaryCtaClick,
+        onSignInClick,
         reminderTracking,
         separateArticleCount,
+        includeSignIn,
```

Comment:
is this a new feature you're adding to the moment banner?

Reply:
Yes, but ... the more I think about it, the less I like it. We do need a`signIn` attribute which will include the sign-in copy and CTA line, which appears in some existing Banners, either following the body copy or following the CTA buttons. 

This fix would allow us to add a dropdown to the RRCP banner variants form for Marketing to add the sign-in line (= easy A/B testing). But I'm now thinking that this is more of a design choice and we should probably set the `signIn` attribute in the JSON that gets fed into the Moment template. If Marketing want banner variations with/without the sign-in line then engineers can create two banners from the template and make them available in the RRCP banner dropdown.

Thoughts?

Reply:
Removed. Will fix when we refactor existing banners which use the sign-in link to use the Moment template